packages/react-spinner/src/components/Spinner/Spinner.test.tsx

```diff
@@ -7,6 +7,11 @@ describe('Spinner', () => {
   isConformant({
     Component: Spinner,
     displayName: 'Spinner',
+    disabledTests: [
+      'component-has-static-classname',
+      'component-has-static-classname-exported',
+      'component-has-static-classnames-object',
```

Comment:
Since you are exporting the classnames object, this test should pass and shouldn't be disabled. Only `component-has-static-classname` should be disabled. Also I think if you disable `component-has-static-classname`, `component-has-static-classname-exported` is skipped.